restapi/configure_weaviate_things_helpers.go

```diff
@@ -0,0 +1,41 @@
+/*                          _       _
+ *__      _____  __ ___   ___  __ _| |_ ___
+ *\ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+ * \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+ *  \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+ *
+ * Copyright © 2016 - 2018 Weaviate. All rights reserved.
+ * LICENSE: https://github.com/creativesoftwarefdn/weaviate/blob/develop/LICENSE.md
+ * AUTHOR: Bob van Luijt (bob@kub.design)
+ * See www.creativesoftwarefdn.org for details
+ * Contact: @CreativeSofwFdn / bob@kub.design
+ */
+
+// Package restapi with all rest API functions.
+package restapi
+
+import "math"
+
+// getLimit returns the maximized limit
+func getLimit(paramMaxResults *int64) int {
+	maxResults := serverConfig.Environment.Limit
+	// Get the max results from params, if exists
+	if paramMaxResults != nil {
+		maxResults = *paramMaxResults
+	}
+
+	// Max results form URL, otherwise max = config.Limit.
+	return int(math.Min(float64(maxResults), float64(serverConfig.Environment.Limit)))
+}
+
+// getPage returns the page if set
+func getPage(paramPage *int64) int {
```

Comment:
This is not specific for things.

Reply:
They are added to these files based on where they are used. But off course, can be moved to the general helper.